### ISSUE TITLE
Allow unlimited bins

### ISSUE THREAD

**Issue by samwyse** [secnot/rectpack#2]
I have a number of STL files that I would like to 3D print. It is simple enough to read a bill of materials and pack the bounding boxes of the objects into a bin the size of my printer's base.  However, I need to guess the number of bins that will be produced.  I'd like a way to produce unlimited bins of a size, instead having to know ahead of time.

**Comment by secnot** [comment]
You can add several bins in one go like this:

```python
...
packer.add_bin(table_width, table_height, count=3000)
packer.pack()
for bin in packer:
    for rect in bin.get_rect_list()
...
```
After packing when you loop over the result, only bins where at least one part is placed are shown.
It is not the most efficient but in my defense I never had to pack more than 100 bins.  I will add a better solution for the next release. Something like this:

```python
packer.add_bin(table_width, table_height, count=0)
```
for an infinite number of bins.

**Comment by samwyse** [comment]
Yeah, I started with a dozen, then doubled it each time I ran out, but that was getting boring.  :)  I've got an add_bin_factory() method that seems to be working; give it a width and height and it produces a limitless supply, after the other bins (if any) are exhausted.  I'll create a pull request in the next day or two.

**Comment by tml** [comment]
The docs say that `packer.add_bin(table_width, table_height,  count=float("inf"))` should give you an infinite number of bins.
